--- app/views/versions/_form.html.erb.orig
+++ app/views/versions/_form.html.erb
@@ -2,6 +2,9 @@
 <p><%= f.text_field :wiki_page_title, :label => :label_wiki_page, :size => 60, :disabled => @project.wiki.nil? %></p>
 <p><%= f.date_field :effective_date, :size => 10 %><%= calendar_for('version_effective_date') %></p>

+<%# Хук для добавления дополнительных полей %>
+<%= call_hook(:view_versions_form_details, { f: f, version: @version }) %>
+
 <p><%= f.select :sharing, @version.allowed_sharings.collect {|v| [format_version_sharing(v), v]} %></p>
 <% if @version.new_record? %>
 <p><%= f.check_box :default_project_version, :label => :field_default_version %></p>